packages/cloudreverb/PKGBUILD

```
@@ -0,0 +1,73 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_name=CloudReverb
+pkgname=${_name,,}
+_plugin_uri='http://github.com/xunil-cloud/$_name'
+pkgver=0.2
+pkgrel=1
+pkgdesc='An algorithmic reverb plugin/app for creating huge, endless spaces and modulated echoes'
+arch=(x86_64 aarch64)
+url='https://github.com/xunil-cloud/CloudReverb'
+license=(MIT)
+depends=(alsa-lib freetype2 glibc gcc-libs)
+makedepends=(cmake webkit2gtk)
+checkdepends=(lv2lint)
+optdepends=(
+  'jack: for running the standalone version with JACKs'
```

I only ever use one JACK at a time :P
```suggestion
  'jack: for running the standalone version with JACK'
```